docker/DOCKER.md

```diff
@@ -0,0 +1,51 @@
+
+
+# Run Waltz
+
+### Pre Requisites
+
+* Docker
+* Postgres DB instance
+
+# Configuration
+
+- Container will use default values to connect Waltz to DB and try to `update` Postgres database and `run` Waltz.
```

Comment:
```suggestion
- The Waltz container will use default values to connect its DB.
- By default it will try to `update` its DB and then `run` Waltz.
```